timed/subscription/templates/notify_accountants_order.html

```diff
@@ -0,0 +1,17 @@
+<b>Aufladung von {{hours_added}} Stunde/n</b>
```

Comment:
I suggest translating the template to english.

Also, nobody is probably ever going to buy a single hour, so no need to use the singular here

Reply:
done